AIO/src/org/aio/activities/banking/ItemReqBanking.java

```diff
@@ -187,6 +191,8 @@ public int execute() throws InterruptedException {
             public int execute() throws InterruptedException {
                 if (!getBank().isOpen()) {
                     getBank().open();
+                } else if (getBank().getWithdrawMode() != org.osbot.rs07.api.Bank.BankMode.WITHDRAW_ITEM) {
```

Comment:
`ItemReq`s can be noted, we shouldn't assume that we always want to `WITHDRAW_ITEM` and not `WITHDRAW_NOTE`.

The logic should be moved into the `getItemReq` function. And we should do something like:

```
if (itemReq.isNoted()) {
    if (getBank().getWithdrawMode() != BankMode.WITHDRAW_NOTE) {
        getBank().enableMode(WITHDRAW_NOTE);
    }
} else if (getBank().getWithdrawMode() != BankMode.WITHDRAW_ITEM) {
    getBank().enableMode(WITHDRAW_ITEM);
}
```